Doc/library/functools.rst

```diff
@@ -208,6 +208,55 @@ The :mod:`functools` module defines the following functions:
       Returning NotImplemented from the underlying comparison function for
       unrecognised types is now supported.
 
+.. function:: topsort(pairs)
```

Comment:
The functions in the functools module are documented in (nearly) alphabetical order, so `topsort` needs to come between `singledispatch` and `update_wrapper`, not here.